CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [2.13.2] - 2024-12-10
```

Comment:
@Harsh517-tech415 can you please remove version and date? @manni497 will re-check soon the pr.. thanks!!